client/app/services/auth.js

```diff
@@ -55,6 +55,8 @@ export const Auth = {
     window.location.href = `${AuthUrls.Login}?next=${next}`;
   },
   logout() {
+    localStorage.removeItem("recentQueries");
+    localStorage.removeItem("recentDashboards");
```

Comment:
I totally forgot about this case of expiration, for sure this need to discussed. I like the backend idea because by this way we can save the recent user data for future sessions, that would be nice.

Reply:
What about reset the recents in localstorage when user login? So the ids that remain in the localstorage don't bother the next user experience if the last user didn't logout by using the logout button.

Reply:
That's a good point, @pxd3v. We wouldn't want to clear it out on login, because this might happen often (depends on various settings) and then recents will be less useful as they will be cleared often.

To address the issue you present (items bothering the next user), we can scope the localStorage item to the user_id and org_id. Something like: `redash.recentQueries.{orgId}.{userId}`.